web-packages/imgui-demo/src/panels/node-graph/visual-graph/base-graph.ts

```
@@ -0,0 +1,841 @@
+    return this.m_childGraph;
+  }
+
+  HasSecondaryGraph (): boolean {
+    return this.m_secondaryGraph !== null;
+  }
+
+  GetSecondaryGraph (): BaseGraph | null {
+    return this.m_secondaryGraph;
+  }
+
+  protected CreateChildGraph<T extends BaseGraph>(constructor: new (...args: any[]) => T, ...params: any[]): T {
```

Biome (1.9.4)

[error] Do not shadow the global "constructor" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)